realm/realm-annotations-processor/src/test/resources/io/realm/BooleansRealmProxy.java

```diff
@@ -25,7 +25,7 @@
 import some.test.Booleans;
 
 public class BooleansRealmProxy extends Booleans
-        implements RealmObjectProxy {
+    implements RealmObjectProxy, BooleansRealmProxyInterface {
```

Comment:
Nothing functional, but it would perhaps be more clear as the purpose of `RealmObjectProxy` is just a marker interface for the proxy classes, which is already implied by RealmProxyInterface.